test/helper/testHelpers.ts

```diff
@@ -33,3 +35,12 @@ export const compareValues = (a: any, b: any): boolean => {
   }
   return a === b;
 };
+
+export const tranferFundsFromCtoP = async () => {
```

Comment:
please add documentation for all the exported function, interfaces, constants and enums